pkg/apis/crd/v1alpha1/types.go

```diff
@@ -600,3 +600,21 @@ type NamespacedName struct {
 	Name      string `json:"name,omitempty"`
 	Namespace string `json:"namespace,omitempty"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type AccountNodeMapping struct {
```

Comment:
Do you think we can check the ServiceAccount has update permission to which Nodes, and validate based on that, instead of adding a new CRD?

Reply:
If we let controller to create and delete AntreaAgentInfo, then agent will only need update AntreaAgentInfo with certain resource name, then admission webhook should not be needed. We can change monitoring part of logics to achieve this. But controller still needs to know when to create AntreaAgentInfo for VM case, as ExternalEntity can also be used as source/destination besides applyTo.